fenix.json.example

```diff
@@ -0,0 +1,25 @@
+{
+    "query": {
+        "bbox": {
+            "n": "-27.2155",
+            "s": "-27.9410",
+            "e": "-48.2711",
+            "w": "-49.0155"
+        }
+    },
+    "agency": {
+        "agency_id": "BR-Floripa",
+        "agency_name": "Consórcio Fênix",
+        "agency_url": "http://www.consorciofenix.com.br/",
+        "agency_timezone": "America/Sao_Paulo",
+        "agency_lang": "pt",
+        "agency_phone": "+55 (48) 3025-6868"
+    },
+    "feed_info": {
+        "publisher_name": "Torsten Grote",
+        "publisher_url": "https://transportr.grobox.de",
+        "version":  "0.1"
+    },
+    "start_date": "20160901",
+    "end_date": "20170831"
```

Comment:
Good idea, but I opened an issue (#14) for this, because i't's a feature request. Right now we first need to get the functionality which is already there and make the script useful for others/us. New features shouldn't be show stoppers in this process.